### DISCUSSION TITLE
How to determine magento db version

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1790] [Questions & Answers] [answered]
Is there something registered in the magento database, that says what version / patch level it is at?

**Comment by pquerner** [comment, marked as answer]
In the database table `core_resource` you can find which version of an extension/module is loaded and which data level (patch level) is already executed from sql setup files.
Read more on that topic here: https://alanstorm.com/magento_setup_resources/

**Reply by f1-outsourcing** [reply]
thanks!
